### ISSUE TITLE
Annotation of dev-305 and train-v2-1166

### ISSUE THREAD

**Issue by nikitakit** [UniversalDependencies/UD_Danish#17]
I've been running scripts to detect rare instances of highly non-projective sentences in UD, and came across these two in DDT. I have zero knowledge of Danish, but I wonder if these might be annotation errors.

For `dev-305`, the analysis of _"flyet med næsen pegende nedad ... ned"_ looks strange.
> "Jeg så flyet med næsen pegende nedad, venstre vinge op og højre vinge ned ovre bag den flade bygning.
<img width="654" alt="dev-305" src="https://github.com/UniversalDependencies/UD_Danish-DDT/assets/252225/e5e1809c-b513-41cf-896e-311b10e9ddc5">

----

For `train-v2-1166` there might not be an error (because other instances are annotated the same way), but the attachment _Men_->_svært_ (as opposed to e.g. _Men_ -> _populær_) makes this tree highly non-projective

> Men bøger er jo ikke så populær som underholdning som tidligere, og det er svært at sige, hvor mange der i dag kender ham her i USA.
<img width="875" alt="train-v2-1166" src="https://github.com/UniversalDependencies/UD_Danish-DDT/assets/252225/bcfa6cf4-2759-4d1b-be2e-069496e9e8a2">

**Comment by KennethEnevoldsen** [comment]
Hi @nikitakit, thanks for creating this issue. I am at a conference at moment so will take a closer look at this at the end of the week - however hopefully you can help me understand the problem a bit better beforehand.

I myself am not a linguist by training, but I do understand Danish so was hoping you could explain why you believe there is an error (eg. What does it mean that it is highly non-projective? I assume that is a bad thing?)

**Comment by martinpopel** [comment]
Non-projective edges (simply: edges crossing other edges) are not bad per se. There are many constructions which need to be annotated non-projectively in UD. That said, some types of non-projectivities are very rare.
I am quite sure the examples above are annotation errors. In `dev-305`: _med_ should depend on  _næsen_, but there are several other errors (not causing non-projectivities).
In `train-v2-1166`, I also think there should be _Men -> populær_. The reason why it is wrong is perhaps because DDT originally used another style of coordination and it was converted automatically to UD. I remember we had several [unfinished issues](https://github.com/ufal/treex/commit/1e6376861ce3d4bd16b1eae71ae1414cc1ffeea7#diff-a8c7756e8a42d863722a55a1b4e52638267dcb078966193ec98563b4cc0ab7a4R33-R35) in the conversion 8 years ago (although I am not sure what is the final version used for converting DDT to UD).

**Comment by nikitakit** [comment]
@martinpopel  Thank you for confirming my suspicions that these are likely annotation errors.

@KennethEnevoldsen While crossing edges are quite common in UD, certain configurations of too many crossing edges are extremely rare or even entirely unattested. I've been looking at UD to try to get a better sense of where the boundary between rare and impossible is, and in the process I've come across a number of trees that are simply annotation errors and not examples of rare syntactic patterns.

**Comment by KennethEnevoldsen** [comment]
Thanks for the explanation both of you. I have made a PR addressing these two cases above. I asked a local linguist who seems to agree (though she isn't as comfortable with this annotation scheme). However, do feel free to grab a look at the PR. However, the reviewers are more familiar and will likely clarify it.

> I remember we had several [unfinished issues](https://github.com/ufal/treex/commit/1e6376861ce3d4bd16b1eae71ae1414cc1ffeea7#diff-a8c7756e8a42d863722a55a1b4e52638267dcb078966193ec98563b4cc0ab7a4R33-R35) in the conversion 8 years ago (although I am not sure what is the final version used for converting DDT to UD).

Feel free to open any issues related to DDT I will be happy to take a look at them

**Comment by nikitakit** [comment]
Thank you for addressing these!
